docker/docker-compose.2004.56.yaml

```diff
@@ -12,6 +12,7 @@ services:
 
   unit-tests:
     image: swift-nio-http2:20.04-5.6
+    command: /bin/bash -xcl "swift test -Xswiftc -warnings-as-errors --enable-test-discovery"
```

Comment:
Rather than doing this override for all the version 5.5+ why don't you change the default and override for 5.4?

Reply:
Giving this a shot now.

Reply:
We need test discovery disabled on 5.5 too because we have deprecated tests here. Otherwise this works fine.